about.html

```diff
@@ -26,6 +26,8 @@
 		<!-- Variable Tags -->
 		<meta property="og:title" content="About Us | TBFighters" />
 		<meta property="og:description" content="Fighting for accessible tuberculosis testing and treatment to save over a million lives a year. Take action with us now." />
+		<script src="./actions.js"></script>
```

Comment:
should this have the defer bit?

Reply:
It should not, `actions.js` contains a bunch of functions to be called elsewhere, `action-page.js` (the one linked to) is the code for the actions on the action page (expand button and slideshow).